src/cnaas_nms/db/settings.py

```diff
@@ -271,8 +271,8 @@ def check_settings_syntax(settings_dict: dict, settings_metadata_dict: dict) ->
                     pydantic_descr_msg = ", field should be: {}".format(pydantic_descr)
                 else:
                     pydantic_descr_msg = ""
-            except Exception as e:
-                logger.debug(e)
+            except Exception as exception:
```

Comment:
Maybe this one could be called descr_exception or something to make it even more clear?

Reply:
Good idea, or maybe descr_error?